ios/RCTWKWebView/RCTWKWebView.m

```diff
@@ -123,7 +123,11 @@ -(void)setHideKeyboardAccessoryView:(BOOL)hideKeyboardAccessoryView
 - (void)userContentController:(WKUserContentController *)userContentController didReceiveScriptMessage:(WKScriptMessage *)message
 {
   if (_onMessage) {
-    _onMessage(@{@"name":message.name, @"body": message.body});
+    NSMutableDictionary<NSString *, id> *event = [self baseEvent];
+    [event addEntriesFromDictionary: @{
+      @"data": message.body,
```

Comment:
`message.name` allow distinguish between message handlers, and probably we should keep it.

Reply:
makes sense, I'll add it to the event.

Reply:
Updated